### DISCUSSION TITLE
Happy holidays! Slow in support.

### DISCUSSION THREAD

**Discussion by knmcguire** [bitcraze/discussions#489] [Announcements]
Mind that we will be slow with reacting on your questions due to the Christmas break. We will be back on track first thing in January so please be patient! 

In the meantime; check out the cool Loco positioning indoor show video we made!  Happy holidays!

https://youtu.be/w8jAHYlcj7k

**Comment by magalhaesw** [comment]
Thanks knmcguire!!
Happy 2023 !!
